src/main/java/com/airlines/dao/impl/CrewMemberDaoImpl.java

```diff
@@ -55,7 +56,7 @@ public void update(CrewMember crewMember) {
         Objects.requireNonNull(crewMember);
 
         if (crewMember.getId() == 0) {
-            throw new DaoOperationException("Cannot find a product without ID");
+            throw new DaoOperationException("Cannot find a crew member without ID");
         }
 
         try (Connection connection = dbConnector.getDBConnection();
```

Comment:
Another example of violation consistency and symmetry pricniples:
For same case - using different approaches:

dbConnector
getDBConnection

In most case, it would be `dbConnector.getDbConnection();`

I didn't comment this because it's opinion based, and I didnt want to interfere my personal preference with your ideas in this case.

But same approach should be used

Reply:
done in https://github.com/pielena/airlines-jdbc-app/commit/80c1f66083580a28c27bea038eea1e050a97f30e